lib/CMakeLists.txt

```diff
@@ -70,6 +70,6 @@ if (Boost_FOUND)
     target_include_directories(cppcheck-core-private SYSTEM PRIVATE ${Boost_INCLUDE_DIRS})
 endif()
 
-if (NOT CMAKE_DISABLE_PRECOMPILE_HEADERS)
-    target_precompile_headers(cppcheck-core-private PRIVATE precompiled.h)
-endif()
+# if (NOT CMAKE_DISABLE_PRECOMPILE_HEADERS)
+#     target_precompile_headers(cppcheck-core-private PRIVATE precompiled.h)
+# endif()
```

Comment:
Needs to be enabled again before merging.

Reply:
Yea, that was to test the CI. I was getting error:

```
../../../lib/CMakeFiles/cppcheck-core-private.dir/cmake_pch.hxx.gch: file not recognized: file format not recognized
```

Now with it disabled I get an error that there isnt a make rule for it.